### ISSUE TITLE
Clutterreport: /storage/emulated/0/.fe_tmp (new)

### ISSUE THREAD

**Issue by M1cha3lWeld0n** [d4rken-org/sdmaid#5948]
### Target
Target: `/storage/emulated/0/.fe_tmp`
Prefix-free: `.fe_tmp`
Path-prefix: `/storage/emulated/0/`
Type: `FILE`
Location: `SDCARD`
Current keeper state: `false`
Suggested keeper state: `false`

### Suggested owners
Appname: `Huawei Share`
Packagename: `com.huawei.android.instantshare`
Version: `9.1.0.101 (90100101)`

### Current owners
None

### SD Maid
Version: `5.5.8 (50508)`

### Device
Fingerprint: `HUAWEI/FIG-LX1/HWFIG-H:9/HUAWEIFIG-L11/9.1.0.176C02:user/release-keys`

**Comment by d4rken** [comment]
Can you share more details on this?

**Comment by M1cha3lWeld0n** [comment]
Current keeper state: false
Suggested keeper state: false

I wondered why both of the above were 'false'?

I'm just starting out using GitHub and trying to get a little understanding of programs, coding etc.. 
I've used terminal/Termux for a while longer from when it was downloadable from the Play Store and so on..
I've never really gone to far as to details? I've rooted a few phones I've had and I'm very interested in gaining root without a PC namely the Huawei device above (FIG-L1X) I've tried many if not all the most recommended ones etc.. like king, kingo, frama and Xsposed/ed.. and no luck if you have any idea on what direction I should take on this that would be great? (And yes another one of those rooting requests I know sorry)

Back to the top post..

And a major issue is that the only document I have available is that? I can't access any settings at all I can't even adjust the display time out which is 15 seconds and an absolute nightmare! I click on the settings app and the main cog on the drop down menu and it flashes a black screen and search bar and then closes and I have no idea what to do about it? All the applications work fine battery life is amazing However which is great but no settings and no access to Dev ops is a killer and I don't want to hard reset it as I am worried it won't start up ever again and I absolutely love this phone I mean I would....... I have built it from a shell which was about all that was left of it I've given it everything it deserves & more it even has a miniature cooling system and led inner lighting which flashes when it rings and barks when I whistle so I don't lose it.. I know sad right? But software not my expertise but I get by?

Thank you
